### PR TITLE
speedy: Align A7 criteria labels with the actual criteria

### PR DESCRIPTION
**Pull request by mdaniels5757**
CSD A7 "applies to any article about a [list of things omitted] that does not indicate why its subject is important or significant". However, currently Twinkle summarizes A7 as "Unremarkable", which isn't accurate -- there are plenty of unremarkable things that do not satisfy the very low bar of A7 (which "does not apply to any article that makes any credible claim of significance or importance even if the claim is not supported by a reliable source or does not qualify on Wikipedia's notability guidelines.")

I propose changing "Unremarkable" to "No indication of importance or significance of subject".

### CODE REVIEW

**Comment by mdaniels5757** [comment]
I agree it's a bit too wordy, and have pushed a change that removes the "of subject" bit (so they are now `No indication of importance or significance (person)`.

I'm not sold dropping "significance" is the best way to shorten it further: "importance" and "significance" are slightly different (at least, different enough to warrant including both words in the actual criteria). If we had to drop one word, I'd argue keeping "significant" and removing "important" stays truer to the original meaning: at least to me, something important is normally significant, but something significant is often not important. (In other words, importance normally implies significance, but significance often does not imply importance)

If you wanted to keep both words and consolidate slightly more, and were ok with using slashes, `No indication of importance/significance (person)` would be an easy change to make that drops 3 characters.

**Comment by Amorymeltzer** [comment]
I suggested "No indication of importance" because that's the actual header used for the criterion on the CSD policy page, viz https://en.wikipedia.org/wiki/Wikipedia:Criteria_for_speedy_deletion#A7._No_indication_of_importance_(people,_animals,_organizations,_web_content,_events).

**Comment by mdaniels5757** [comment]
Yup, you're right. Done.
